libethereum/Client.cpp

```diff
@@ -377,18 +381,27 @@ void Client::appendFromBlock(h256 const& _block, BlockPolarity _polarity, h256Ha
     }
 }
 
-unsigned static const c_syncMin = 1;
-unsigned static const c_syncMax = 1000;
-double static const c_targetDuration = 1;
-
 void Client::syncBlockQueue()
 {
 //  cdebug << "syncBlockQueue()";
 
     ImportRoute ir;
     unsigned count;
     Timer t;
-    tie(ir, m_syncBlockQueue, count) = bc().sync(m_bq, m_stateDB, m_syncAmount);
+
+    shared_ptr<VerifiedBlocks> verifiedBlocks = make_shared<VerifiedBlocks>();
```

Comment:
shared_ptr is required here since we propagate the blocks on another thread so we need to ensure that they are kept around